olm-catalog/serverless-operator/1.6.0/serverless-operator.v1.6.0.clusterserviceversion.yaml

```
@@ -68,6 +68,7 @@ metadata:
       Provides a collection of API's based on Knative to support deploying and serving
       of serverless applications and functions.
     repository: https://github.com/openshift-knative/serverless-operator
+    operators.operatorframework.io/internal-objects: '["knativeservings.operator.knative.dev", "knativeservings.serving.knative.dev"]'
```

```suggestion
    operators.operatorframework.io/internal-objects: '["knativeservings.operator.knative.dev"]'
```

The old group no longer exists.